package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "zestaw-7",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview",
+    "lint": "eslint src/**/*.{ts,tsx}",
+    "lint:fix": "eslint --fix src/**/*.{ts,tsx}",
+    "format": "prettier --write src/**/*.{js,jsx,ts,tsx,css,md,json,scss} --config ./.prettierrc"
+  },
+  "lint-staged": {
+    "**/*.{ts,tsx}": [
+      "npm run lint"
+    ]
+  },
+  "dependencies": {
+    "@chakra-ui/react": "^2.8.2",
+    "@supabase/supabase-js": "^2.39.3",
+    "formik": "^2.4.5",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "yup": "^1.3.3"
+  },
+  "devDependencies": {
+    "@commitlint/cli": "^18.6.0",
```

Comment:
co tym cli robisz?

Reply:
nie wiem, moze cos probowalem ogarnac jak wywalalo mi husky.

Reply:
Nieużywane paczki też zawsze do wywalenia

Reply:
jak to usuwac? jak to kontrolowac?